perun-openapi/openapi.yml

```diff
@@ -13074,10 +13074,10 @@ paths:
               type: object
               required:
                 - mailType
-                - applicationId
+                - appId
               properties:
                 mailType: { $ref: '#/components/schemas/MailType' }
-                applicationId: { type: integer, description: "application id" }
+                appId: { type: integer, description: "application id" }
```

Comment:
Parameter didn't change, it was wrong here all the time. But there is a generic (now unused) parameter ``applicationId`` at the top of the file. Maybe there is switched ``appId`` and ``applicationId`` in it. If fixed we could use its definition from here and also other methods.